components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/notifier/events/ScopesEvent.java

```diff
@@ -0,0 +1,25 @@
+package org.wso2.carbon.apimgt.impl.notifier.events;
```

Comment:
Missing licence headers

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/12090/commits/4cfe1d02686745ba8cec054c183f3f74d835dd64